frontend/src/concepts/areas/const.ts

```diff
@@ -0,0 +1,45 @@
+import { StackComponent, SupportedArea, SupportedAreasState } from './types';
+
+export const SupportedAreasStateMap: SupportedAreasState = {
+  [SupportedArea.BYON]: {
+    featureFlags: ['disableBYONImageStream'],
+  },
+  [SupportedArea.CLUSTER_SETTINGS]: {
+    featureFlags: ['disableClusterManager'],
+  },
+  [SupportedArea.CUSTOM_RUNTIMES]: {
+    featureFlags: ['disableModelServing', 'disableCustomServingRuntimes'],
+    reliantAreas: [SupportedArea.MODEL_SERVING],
+  },
```

Comment:
Does this require `disableModelServing` feature flag if it's reliant on area MODEL_SERVING?

Reply:
I sorta just configured it to be what would actually be needed -- but you are right, it doesn't -- reliant areas were not there in first pass.